src/shipit_static_analysis/shipit_static_analysis/cli.py

```diff
@@ -12,10 +12,8 @@
 from cli_common.log import init_logger
 from cli_common.taskcluster import get_secrets
 import click
-import logging
 import re
 
-logger = logging.getLogger(__name__)
```

Comment:
As far as I can tell this is just leftover code, and doesn't actually need replacement by cli_common.log